tycho/test/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@mbrightman I believe this file should also be removed from the commit since it seems to be an editor config file?

Reply:
I agree. Will remove it!